Example/PINRemoteImage Tests/PINRemoteImage_Tests.m

```diff
@@ -10,7 +10,9 @@
 #import <XCTest/XCTest.h>
 #import <PINRemoteImage/PINRemoteImage.h>
 #import <PINRemoteImage/UIImageView+PINRemoteImage.h>
+#if __has_include(<FLAnimatedImage/FLAnimatedImage.h>)
```

Comment:
Yeah, I can change that, that makes sense. I guess I had a logical separation where I thought that the sole responsibility of `USE_FLANIMATED_IMAGE` was the macro for defining the object type of `FLAnimatedImage`.